e2e/LongHaul/prerequisites/LongHaulSetup.ps1

```diff
@@ -295,11 +285,11 @@ foreach ($kvp in $keyvaultKvps.GetEnumerator())
 # Creating a file to run to load environment variables
 ############################################################################################################################
 
-$loadScriptDir = Join-Path $PSScriptRoot "..\..\..\.." -Resolve
-$loadScriptName = "Load-$keyVaultName.ps1";
+$loadScriptDir = Join-Path $PSScriptRoot "..\..\.." -Resolve
+$loadScriptName = "Load-LongHaul-$keyVaultName.ps1";
 Write-Host "`nWriting environment loading file to $loadScriptDir\$loadScriptName.`n"
 $file = New-Item -Path $loadScriptDir -Name $loadScriptName -ItemType "file" -Force
-Add-Content -Path $file.PSPath -Value "$loadScriptDir\azure-iot-sdk-csharp\e2e\Tests\prerequisites\LoadEnvironmentVariablesFromKeyVault.ps1 -SubscriptionId $SubscriptionId -KeyVaultName $keyVaultName"
```

Comment:
People might have renamed their root folder name, so we shouldn't take a dependency on that